fetch.bs

```diff
@@ -6680,6 +6720,308 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+     <a>active document</a> is not <a>allowed to use</a> <var>containerPolicy</var>, then return 0.
+
+     <li><p>Otherwise, set <var>remainingTotalQuota</var> to the
+     <a>initial subframe deferred fetch quota</a> given <var>containerPolicy</var>.
+    </ol>
+  </ol>
+
+ <li><p>Let <var>remainingQuotaForOrigin</var> be <var>remainingTotalQuota</var>.
+
+ <li><p>If <var>remainingQuotaForOrigin</var> is greater than 64 kibibytes, then set
+ <var>remainingQuotaForOrigin</var> to 64 kibibytes.
+
+ <li>
+  <p><a for=list>For each</a> <var>navigable</var> in <var>relativeNavigables</var>:
+  <ol>
+   <li>
+    <p><a for=list>For each</a> <a>deferred fetch record</a> <var>deferredRecord</var> in
+    <var>navigable</var>'s <a>active document</a>'s <a for=fetch>fetch group</a>'s
+    <a for="fetch group">deferred fetch records</a>:</p>
+
+    <ol>
+     <li><p>Let <var>length</var> be <var>deferredRecord</var>'s
+     <a for="deferred fetch record">request</a>'s <a for=request>body</a>'s
+     <a for=body>length</a>.
+
+     <li><p>Decrement <var>remainingTotalQuota</var> by <var>length</var>.
+
+     <li><p>If <var>origin</var> is not null, and <var>deferredRecord</var>'s
+     <a for="deferred fetch record">request</a>'s <a for=request>URL</a>'s <a for=url>origin</a> is
+     <a>same origin</a> with <var>origin</var>, then decrement <var>remainingQuotaForOrigin</var> by
+     <var>length</var>.
+    </ol>
+   </li>
+
+   <li><p>Let <var>relevantContainers</var> be « ».
+   <li><p><a for=list>For each</a> <var>relative</var> in <var>navigable</var>'s
+   <a>deferred fetch quota-sharing navigables</a>, <a for=list>extend</a>
+   <var>relevantContainers</var> with every <a>shadow-including descendant</a> of
+   <var>relative</var> which is a <a>navigable container</a>.
+
+   <li><p><a for=list>For each</a> <var>descendantContainer</var> of <var>relevantContainers</var>,
+   decrement <var>remainingTotalQuota</var> by the <a>initial subframe deferred fetch quota</a>
+   given <var>descendantContainer</var>'s <a>deferred fetch policy</a>.
+  </ol>
+ </li>
+
+ <li><p>If <var>remainingTotalQuota</var> is less than <var>remainingQuotaForOrigin</var>, then
+ return <var>remainingTotalQuota</var>.
+
+ <li><p>Otherwise, return <var>remainingQuotaForOrigin</var>.
+</ol>
+</div>
+
+<div algorithm>
+<p>To <dfn export>determine subframe deferred fetch policy</dfn> for a <a>navigable container</a>
+<var>container</var> and an <a for=/>origin</a> <var>originToNavigateTo</var>:
+
+<p class=note>This is called whenever a <a>navigable container</a> is being navigated, e.g. by
+setting an iframe's <a attribute for=HTMLIFrameElement>src</a>.
+
+<ol>
+ <li><p>Set <var>container</var>'s <a>deferred fetch policy</a> to <code>disabled</code>.
+
+ <li><p>If the <a data-lt="define an inherited policy for feature in container">inherited policy</a>
+ for {{PermissionPolicy/"deferred-fetch"}}, <var>navigable</var>'s <a>navigable container</a> and
```

Comment:
```
If the inherited policy for "deferred-fetch", navigable’s navigable container and originToNavigateTo is Enabled,
```

Does `navigable’s navigable container` mean the same `container` provided to [determine subframe deferred policy](https://whatpr.org/fetch/1647.html#determine-subframe-deferred-fetch-policy)? If so, can't it just be rephrased as `container`?

```
then set navigable’s deferred fetch policy to "deferred-fetch" and return.
```

Does this step try to set `navigable`'s deferred fetch policy? If not, should it just be `container`?

Reply:
FIxed